0_Fib/fib.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func Fibonacci(n int) []int {
+
+	fib, t1, t2 := []int{}, 1, 1
+	for i := 0; i < n; i++ {
+		// add the next member
+		fib = append(fib, t1)
+		t1, t2 = t2, t1+t2
+	}
+	return fib
+}
+func arrayToString(a []int, delim string) string {
```

Comment:
This can be replaced by std lib https://golang.org/pkg/strings/#Join

Reply:
done